massmail

```diff
@@ -114,14 +111,15 @@ def error(s):
 def parse_command_line_options(arguments):
     # parse options
     try:
-        opts, args = getopt.getopt(arguments, "hfs:F:S:B:R:e:u:p:P:z:")
+        opts, args = getopt.getopt(arguments, "hfs:F:S:B:R:e:u:p:P:z:send:")
```

Comment:
I think this should read `opts, args = getopt.getopt(arguments, "hfs:F:S:B:R:e:u:p:P:z:", ['send']) `